Makefile

```diff
@@ -151,9 +151,14 @@ bindings: .create-bindings .strip-autogen-comment .strip-nvml-h-linenumber
 		go fmt types_gen.go; \
 	cd -> /dev/null
 	rm -rf $(PKG_BINDINGS_DIR)/nvml.yml $(PKG_BINDINGS_DIR)/cgo_helpers.go $(PKG_BINDINGS_DIR)/types.go $(PKG_BINDINGS_DIR)/_obj
-	go run $(GEN_BINDINGS_DIR)/generateapi.go \
+	go run -tags generateapi \
```

Comment:
OK, so we're using build tags to select between the two different `main` packages. Does putting these in their own folders instead simplify things?

Reply:
I guess we could, but I really want to keep each of them as their own standalone scripts without an attempt to share code between them (meaning each would likely be a folder with a single file each in them).